kubernetes/client/configuration.py

```diff
@@ -235,3 +224,11 @@ def to_debug_report(self):
                "Version of the API: v1.5.0-snapshot\n"\
                "SDK Package Version: 1.0.0-snapshot".\
                format(env=sys.platform, pyversion=sys.version)
+
+
+configuration = ConfigurationObject()
+
+
+def Configuration():
```

Comment:
Who is the caller of this function? If a client is created via `new_client_from_config`, I think `configuration` will contain default values rather than values from a real kube config file, is that the callers of `Configuration()` expect?